htdocs/compta/facture/prelevement.php

```diff
@@ -803,24 +799,26 @@
 				//print '</td>';
 				//print '<td class="left nowraponall">';
 
-				$ribList = $object->thirdparty->get_all_rib();
-				$ribForSelection = [];
-				$defaultRib = '';
-				foreach ($ribList as $rib) {
-					$ribString = $rib->iban . (($rib->iban && $rib->bic) ? ' / ' : '') . $rib->bic;
-					$ribForSelection[$rib->id] = $ribString;
-					if ($rib->default_rib == 1) {
-						$defaultRib = $ribString;
+				// if societe rib in model invoice, we preselect it
+				$selectedRib = '';
+				if ($object->element = 'invoice' && $object->fk_fac_rec_source) {
```

Comment:
You surely want to do 
`$object->element == 'invoice'`
instead of
`$object->element = 'invoice'`

Reply:
Oups ! It's done